Spigot-API-Patches/0067-Added-ban-events-resolves-678.patch

```diff
@@ -0,0 +1,151 @@
+From ee66380f6bf8892079012b6f225fad34b64bcc66 Mon Sep 17 00:00:00 2001
+From: Sam Ezeh <bubuezeh@gmail.com>
+Date: Wed, 30 Aug 2017 21:32:18 +0100
+Subject: [PATCH] Added ban events,  resolves #678
+
+
+diff --git a/src/main/java/org/bukkit/event/server/BanEvent.java b/src/main/java/org/bukkit/event/server/BanEvent.java
+new file mode 100644
+index 00000000..dcd95bf6
+--- /dev/null
++++ b/src/main/java/org/bukkit/event/server/BanEvent.java
```

Comment:
if there is anything we have learnt, any classes we add should ideally be in our own package as opposed to bukkits